### DISCUSSION TITLE
Documentation for Mitsubishi HVAC Module

### DISCUSSION THREAD

**Discussion by host-felice** [arendst/Tasmota#22393] [Q&A]
Hello,
I’m currently working with the Mitsubishi HVAC module and looking to understand more about its specific commands and functionality, especially working with HVACRemoteTemp command. However, I haven't been able to locate any documentation that goes into detail about the Mitsubishi HVAC controls.

Could you let me know if any documentation is available for this module? If it’s not yet documented, any examples would be appreciated!

Thank you for your assistance!

**Comment by echo-bravo-yahoo** [comment]
if you're still looking, i have a functional HVACRemoteTemp setup.

a remote device sends MQTT messages to the HVAC module's topic with the temperature that the remote device senses. in my case, another device is publishing to `cmnd/destination/HVACRemoteTemp degreesC` (which maps to running `HVACRemoteTemp degreesC` in the web console of the HVAC module).

the only caveat here is that you need to set `HVACRemoteTempClearTime`, which is the number of ms the HVAC device waits before swapping back to its internal temperature sensor. you'll want it to be _at least_ as long as your publisher's delay between messages. i have it set up to publish every minute, and fall back to the internal sensor every 5 minutes (`HVACRemoteTempClearTime 300000`).

**Comment by host-felice** [comment]
hi @echo-bravo-yahoo thank you very much for your reply and explaination. Do you know any other commands? I saw the HVACRemoteTemp clean. It is a pity we don't have any documentation... how should we know about them?

**Reply by echo-bravo-yahoo** [reply]
I don't have any real insight into this - I read the source code, other discussions, and watched the console log output to guess how to put the commands together. Let me know if you have any other questions I might be able to answer.
